sdk/samples/iot/paho_iot_provisioning_sas_sample.c

```diff
@@ -94,33 +93,32 @@ static void create_and_configure_mqtt_client(void)
   int rc;
 
   // Reads in environment variables set by user for purposes of running sample.
-  if (az_result_failed(
-          rc = iot_sample_read_environment_variables(SAMPLE_TYPE, SAMPLE_NAME, &env_vars)))
+  rc = iot_sample_read_environment_variables(SAMPLE_TYPE, SAMPLE_NAME, &env_vars);
+  if (az_result_failed(rc))
   {
-    IOT_SAMPLE_LOG_ERROR(
-        "Failed to read configuration from environment variables: az_result return code 0x%08x.",
-        rc);
+    IOT_SAMPLE_LOG_ERROR("Failed to read environment variables: az_result return code 0x%08x.", rc);
     exit(rc);
   }
 
   // Build an MQTT endpoint c-string.
   char mqtt_endpoint_buffer[256];
-  if (az_result_failed(
-          rc = iot_sample_create_mqtt_endpoint(
-              SAMPLE_TYPE, &env_vars, mqtt_endpoint_buffer, sizeof(mqtt_endpoint_buffer))))
+  rc = iot_sample_create_mqtt_endpoint(
+      SAMPLE_TYPE, &env_vars, mqtt_endpoint_buffer, sizeof(mqtt_endpoint_buffer));
+  if (az_result_failed(rc))
   {
     IOT_SAMPLE_LOG_ERROR("Failed to create MQTT endpoint: az_result return code 0x%08x.", rc);
     exit(rc);
   }
 
-  // Initialize the provisioning client with the mqtt endpoint and the default connection options.
-  if (az_result_failed(
-          rc = az_iot_provisioning_client_init(
-              &provisioning_client,
-              az_span_create_from_str(mqtt_endpoint_buffer),
-              env_vars.provisioning_id_scope,
-              env_vars.provisioning_registration_id,
-              NULL)))
+  // Initialize the provisioning client with the provisioning global endpoint and the default
+  // connection options.
+  rc = az_iot_provisioning_client_init(
+      &provisioning_client,
+      az_span_create_from_str(mqtt_endpoint_buffer),
+      env_vars.provisioning_id_scope,
+      env_vars.provisioning_registration_id,
+      NULL);
+  if (az_result_failed(rc))
```

Comment:
nit: Consider adding a new line between rc returning operation and the error check (everywhere). Your call.

```suggestion
      NULL);

  if (az_result_failed(rc))
```